playgrounds/geo-javascript/.env

```diff
@@ -1 +0,0 @@
-GOOGLE_API=AIzaSyDOaUaar4GL0i99LpN2zQHzfWXL1wu_JQo
```

Comment:
did we revoke this api key?

Reply:
No, but it's been public for years now. 

And since this API key is intended for use in client-side JavaScript anyway, it's safe to share imo.

That said, @curquiza, if you want to revoke + recreate, you can.